doc/release-notes/release-notes-2.0.6.md

```diff
@@ -0,0 +1,102 @@
+Notable changes
+===============
+
+Debian Stretch is now a Supported Platform
+------------------------------------------
+
+We now provide reproducible builds for Stretch as well as for Jessie.
+
+
+Fixed a bug in ``z_mergetoaddress``
+-----------------------------------
+
+We fixed a bug which prevented users sending from ``ANY_SPROUT`` or ``ANY_SAPLING``
+with ``z_mergetoaddress`` when a wallet contained both Sprout and Sapling notes.
+
+
+Insight Explorer
+----------------
+
+We have been incorporating changes to support the Insight explorer directly from
+``zcashd``. v2.0.6 includes the first change to an RPC method. If ``zcashd`` is
+run with the flag ``--insightexplorer``` (this requires an index rebuild), the
+RPC method ``getrawtransaction`` will now return additional information about
+spend indices.
+
+Changelog
+=========
+
+Charlie O'Keefe (1):
+      Add stretch to list of suites in gitian linux descriptors
+
+Daira Hopwood (9):
+      Closes #3992. Remove obsolete warning message.
+      make-release.py: Versioning changes for 2.0.6-rc1.
+      make-release.py: Updated manpages for 2.0.6-rc1.
+      make-release.py: Updated release notes and changelog for 2.0.6-rc1.
+      ld --version doesn't work on macOS.
+      Tweak author aliases.
+      Add coding declaration to zcutil/release-notes.py
+      make-release.py: Versioning changes for 2.0.6.
+      make-release.py: Updated manpages for 2.0.6.
+
+Daniel Kraft (1):
+      Add some const declarations where they are appropriate.
+
+Eirik Ogilvie-Wigley (1):
```

Comment:
I had to change somethings around on my computer. I wonder if I have done something wonky that makes me show up twice in this list or if it something to do with how the script works.

Reply:
I suspect a bug in the script. I'll investigate that too, but it does not block the release.